content/pt-br/shift-left.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Shift Left
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Esquerda em Shift Left se refere aos estágios iniciais no ciclo de desenvolvimento de software,
+pensando no ciclo como uma linha onde os estágios são executados da esquerda para a direita.
+Shift left é a pratica de implementar testes, segurança e outras praticas de desenvolvimento
```

Comment:
"é a prática de implementar testes, segurança ou outras práticas" instead of "é a pratica de implementar testes, segurança e outras praticas"